index.js

```diff
@@ -32,12 +32,14 @@ var DebugLogtron = require('debug-logtron');
 
 var console = require('console');
 var process = require('process');
-var util = require('util');
 var fs = require('fs');
 var path = require('path');
 var url = require('url');
 var assert = require('assert');
 
+var log = require('./log');
```

Comment:
Whether to use `.js` suffix is a matter of style. @Raynos favors it so that you can follow a file name in a text editor. Prevailing style is to leave it off. We aren’t consistent about either on more than a per-file or sometimes per-project basis.